rules/linux/persistence_linux_group_creation.toml

```diff
@@ -0,0 +1,79 @@
+[metadata]
+creation_date = "2023/02/13"
+integration = ["system"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/03/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies attempts to create a new group. Attackers may create new groups to escalate privileges 
+or establish persistence on a system or domain.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-system.auth-*"]
```

Comment:
Would be nice to test if this works with auditbeat

Reply:
Currently working on checking endgame/auditbeat support for all 4 rules. Will be added in the commit